Pod/Classes/MenuView.swift

```diff
@@ -85,21 +85,21 @@ public class MenuView: UIScrollView {
     
     // MARK: - Lifecycle
     
-    internal init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
+    public init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
```

Comment:
I'll wait for the feature then. No need to be sorry. Thanks for maitaining this project. Save me a lot of time. :)

On Mon, May 16, 2016 at 01:35, Yusuke Kita <notifications@github.com> wrote:
In Pod/Classes/MenuView.swift [https://github.com/kitasuke/PagingMenuController/pull/207#discussion_r63311602] :

> @@ -85,21 +85,21 @@ public class MenuView: UIScrollView {
>      
>      // MARK: - Lifecycle
>      
> -    internal init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {
> +    public init(menuItemTitles: [String], menuItemDescs: [String], options: PagingMenuOptions) {


It should be done soon, but I can merge the multiline text part only first!
Well, its safer for you to wait the feature though.
Sorry for my inconvenience.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub [https://github.com/kitasuke/PagingMenuController/pull/207/files/c483b9856013ae6cac0e453cc968204688739f0f..ce9fb443ed31ea355d421b40bf63ee7ad1726385#r63311602]